test/unit/mdc-list/foundation.test.js

```diff
@@ -640,6 +632,66 @@ test('#handleKeydown bail out early if event origin doesnt have a mdc-list-item
   td.verify(preventDefault(), {times: 0});
 });
 
+test('#focusNextElement focuses next list item and returns that index', () => {
+  const {foundation, mockAdapter} = setupTest();
+
+  td.when(mockAdapter.getListItemCount()).thenReturn(4);
+
+  assert.equal(3, foundation.focusNextElement(2));
+  td.verify(mockAdapter.focusItemAtIndex(3), {times: 1});
+});
+
+test('#focusNextElement focuses first list item when focus is on last list item when wrapFocus=true and returns that ' +
+    'index', () => {
+  const {foundation, mockAdapter} = setupTest();
+
+  td.when(mockAdapter.getListItemCount()).thenReturn(4);
+  foundation.setWrapFocus(true);
+
+  assert.equal(0, foundation.focusNextElement(3));
+  td.verify(mockAdapter.focusItemAtIndex(0), {times: 1});
+});
+
+test('#focusNextElement retains the focus on last item when wrapFocus=false and returns that index', () => {
+  const {foundation, mockAdapter} = setupTest();
+
+  td.when(mockAdapter.getListItemCount()).thenReturn(4);
+  foundation.setWrapFocus(false);
+
+  assert.equal(3, foundation.focusNextElement(3));
+  td.verify(mockAdapter.focusItemAtIndex(3), {times: 0});
```

Comment:
Change 3 to `td.matchers.isA(Number)`

(I suspect either the `3` here or in the equivalent `focusPrevElement` test below is a copypasta)